doc/developer-guides/sw_design_guidelines.rst

```diff
@@ -77,27 +77,10 @@ below:
 Architecture Level
 ==================
 
-Fulfillment to FuSa Standards
------------------------------
-
-The fulfillment matrix of error detection and error handling on architecture
-level to FuSa Standards [IEC_61508-7]_ in ACRN hypervisor is shown in
-:numref:`fulfillment_matrix_arch_level` below.
-
-.. table:: Fulfillment to FuSa Standards on Architecture Level
-   :align: center
-   :widths: auto
-   :name: fulfillment_matrix_arch_level
-
-   +-----------------------+---------------+----------+--------------------------------------+
-   | Technique/Measure     | Ref in        | SIL3     | Company Solution                     |
-   |                       | IEC 61508-7   |          |                                      |
-   +=======================+===============+==========+======================================+
-   | Failure assertion     | C.3.3         | R        | This technique will be used.         |
-   | programming           |               |          | Plan to do range check in hypercalls |
-   |                       |               |          | and HW capability checks.            |
-   +-----------------------+---------------+----------+--------------------------------------+
+Related Requirements of FuSa Standards
+--------------------------------------
```

Comment:
Rather than delete all of this, can we say:
```
Failure assertion programming is used for error detection
and error handling as recommended by FuSa Standards. 
The ACRN hypervisor uses HW capability checks as well as
range checks in hypercalls.
```

Reply:
Hi David, Thanks for your comments, failure assertion programming is defined by IEC 61508-7, please refer C3.3, different person have different point about failure assertion programming, i will confuse some person that "Failure assertion programming is used for error detection
and error handling as recommended by FuSa Standards.". Can put this detail into the official safety plan and just keep reference here for open source?

Reply:
My concern is you have to pay to read these IEC documents.  Previously you wrote that "The ACRN hypervisor uses HW capability checks as well as range checks in hypercalls."  This mention won't be in the IEC documents, so I was thinking you'd want to mention it here.  Maybe @hwang37 has a recommendation?

Reply:
Just plan to move fulfillment table into safety plan, if safety plan is ready, we maybe refer safety plan directly. I will update related description which is not in the IEC standard. Thanks!